key-cert-provisioner/test-signer/Dockerfile

```diff
@@ -13,11 +13,12 @@
 # limitations under the License.
 FROM scratch AS source
 
+ARG CALICO_BASE
```

Comment:
Is this correct to put `CALICO_BASE` arg in the `source` stage instead of at the top level of this file?

Reply:
Moved to the top now